linkage-monitor/src/main/java/com/google/cloud/tools/dependencies/linkagemonitor/LinkageMonitor.java

```diff
@@ -193,27 +203,60 @@ private static void printUsageAndDie() {
         String versionlessCoordinates = model.getGroupId() + ":" + model.getArtifactId();
         artifactToVersion.put(versionlessCoordinates, model.getVersion());
         logger.fine("Found local artifact: " + model);
-        DependencyManagement dependencyManagement = model.getDependencyManagement();
-        if ("pom".equals(model.getPackaging()) && dependencyManagement != null) {
-          // Read the content of a BOM.
-          for (org.apache.maven.model.Dependency dependency :
-              dependencyManagement.getDependencies()) {
-            String managedDependencyVersionlessCoordinates =
-                dependency.getGroupId() + ":" + dependency.getArtifactId();
-            artifactToVersion.put(managedDependencyVersionlessCoordinates, dependency.getVersion());
-            logger.fine("Found local artifact in the BOM: " + dependency);
-          }
-        }
```

Comment:
This reverts https://github.com/GoogleCloudPlatform/cloud-opensource-java/pull/1920, which causes the problem of mixing BOMs and parent POMs.